case.jsonld

```diff
@@ -21,6 +21,35 @@
         "kb": "http://example.org/kb/"
     },
     "uco-core:description": "An Example Case File",
+    "uco-core:UcoInherentCharacterizationThing": [
```

Comment:
`uco-core:UcoInherentCharacterizationThing` is a class, not a property.

I'll add a revision patch.

Reply:
Revised.  This required some moderate scope-creep of the PR, so I probably shouldn't be the last review before merge.